stubs/idiokitHTTP/idiokitHTTP.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import idiokit
+from idiokit import socket, ssl
+
+@idiokit.stream
+def client(host, port):
+    sock = socket.Socket()
```

Comment:
Consider using `with socket.Socket() as sock` here so the socket is closed in the end.